nomt/src/merkle/worker.rs

```diff
@@ -169,14 +164,20 @@ fn warm_up_phase<H: HashAlgorithm>(
             if index == finish_idx {
                 match finish_rx.try_recv() {
                     Err(TryRecvError::Empty) => continue,
-                    Err(e) => anyhow::bail!(e),
+                    Err(e) => panic!(
+                        "Wamr-Up worker, unexpected failure of the finish channel: {:?}",
```

Comment:
```suggestion
                        "Warm-up worker, unexpected failure of the finish channel: {:?}",
```

Reply:
Although, I don't think we should print `{:?}` here, because we do know what it is.